src/js/components/commits/list.tsx

```diff
@@ -100,9 +102,22 @@ const CommitList = ({ commits }: { commits: Commit[] }) =>
       <h2
         className="slds-text-heading_medium
           slds-m-top_large
-          slds-m-bottom_x-small"
+          slds-m-bottom_x-small
+          slds-is-relative"
       >
         Commit History
+        <TourPopover
```

Comment:
@mirisuzanne looking for a second opinion here, but I think section heading elements only accept phrasing content, not divs. Is that correct? 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

![image](https://user-images.githubusercontent.com/1581694/119557585-fd109100-bd65-11eb-9985-767ba4c29c90.png)